### PR TITLE
Ensure dependencies

### PR DESCRIPTION
**Pull request by Elkrival**
This pr updates and removes dependencies, the criteria being if they are being used in the project.  There are dependencies like lodash that implemented specific logic, this logic has been updated with javascript.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -51,15 +47,6 @@ if (process.env.NODE_ENV === 'development') {
 /** favicon setup */
 app.use(favicon(path.join(__dirname, '../public/img/favicon.ico')))
 
-/** security setup */
-app.use(
-  helmet({
```
Comment (mikestone14) [inline server/app.js:57]: should we leave this in? We discussed either removing it or keeping it.

Comment (JackVCurtis) [reply server/app.js:57]: IMO we should keep this, it fills the same role that Rails handles as part of the framework: https://guides.rubyonrails.org/v4.1/security.html#default-headers

File: server/data_processors/DashboardDataProcessor/index.js [hunk 2]
```diff
@@ -39,10 +37,10 @@ class DashboardDataProcessor {
         },
         []
       )
-
-      const max = _.maxBy(variableData, variable)?.[variable]
-      const min = _.minBy(variableData, variable)?.[variable]
-      const mean = _.meanBy(variableData, variable)
+      const max = Math.max(...variableData.map((data) => data[variable]))
```
Comment (JackVCurtis) [inline server/data_processors/DashboardDataProcessor/index.js:40]: 🎉

File: server/app.js (hunk 1 above)
Comment (Elkrival) [reply server/app.js:57]: [b0ab09e](https://github.com/AMP-SCZ/dpdash/pull/518/commits/b0ab09ecac96b308e778e45f782c00d3d1fc2f6a)

**Comment by mikestone14** [comment]
@Elkrival looks like there are still some references to moment in the test files which is making CI fail.

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**